src/nft/components/sell/select/SelectPage.tsx

```diff
@@ -356,6 +513,88 @@ const SelectAllButton = () => {
   )
 }
 
+const CollectionFiltersRow = ({
+  collections,
+  collectionFilters,
+  setCollectionFilters,
+  clearCollectionFilters,
+}: {
+  collections: WalletCollection[]
+  collectionFilters: Array<string>
+  setCollectionFilters: (address: string) => void
+  clearCollectionFilters: Dispatch<SetStateAction<void>>
+}) => {
+  const getCollection = (collectionAddress: string) => {
+    return collections?.find((collection) => collection.address === collectionAddress)
+  }
+  return (
+    <Row paddingTop="18" gap="8" flexWrap="wrap">
+      {collectionFilters &&
+        collectionFilters.map((collectionAddress, index) => (
+          <CollectionFilterItem
+            collection={getCollection(collectionAddress) ?? ({} as WalletCollection)}
```

Comment:
they do not seem to like casting `{} as type` in this codebase so can you just make it `CollectionFilterItem` can you just check if `getCollection` returns a collection and if it doesn't don't render anything?